sdk/src/store.rs

```diff
@@ -32,7 +32,7 @@ use crate::{
     cose_validator::verify_cose,
     jumbf_io::{
         get_file_extension, get_supported_file_extension, is_bmff_format, load_jumbf_from_file,
```

Comment:
We should probably do one more thing in this function.  If we remove the manifest we should clear the XMP.  It will be cleared for the remote case but not the sidecar case.

Reply:
I can add that in a different PR as to not hold this up.

Reply:
good point. We might need to check if there was something we needed to delete in the case where xmp_write is disabled, otherwise we could return an error when there's nothing wrong.

We also need to figure out how to remove c2pa jumbf from in memory buffers. But that can also wait for later.